randovania/data/json_data/prime1.txt

```diff
@@ -1279,379 +2367,833 @@ Asset id: 1125030300
-> Dock to Central Dynamo; Heals? False
-  * Normal Door to Central Dynamo/Dock to Dynamo Access
+> Dock to Central Dynamo; Heals? False; Spawn Point
+  * Ice Door to Central Dynamo/Dock to Dynamo Access
+  > Dock to Omega Research
+      Trivial
 
 > Dock to Omega Research; Heals? False
-  * Normal Door to Omega Research/Dock to Dynamo Access
+  * Ice Door to Omega Research/Dock to Dynamo Access
+  > Dock to Central Dynamo
+      Trivial
 
 ----------------
 Transport to Magmoor Caverns South
 Asset id: 3804417848
 > Dock to Transport Access; Heals? False
-  * Normal Door to Transport Access/Dock to Transport to Magmoor Caverns South
+  * Ice Door to Transport Access/Dock to Transport to Magmoor Caverns South
+  > Elevator - Transport to Phazon Mines West
+      Scan Visor
+
+> Elevator - Transport to Phazon Mines West; Heals? False; Spawn Point
+  * Teleporter to Magmoor Caverns - Transport to Phazon Mines West
+  > Dock to Transport Access
+      Trivial
 
 ----------------
 Elite Quarters
 Asset id: 961790803
-> Dock to Elite Quarters Access; Heals? False
-  * Normal Door to Elite Quarters Access/Dock to Elite Quarters
+> Dock to Elite Quarters Access; Heals? False; Spawn Point
+  * Plasma Door to Elite Quarters Access/Dock to Elite Quarters
+  > Dock to Processing Center Access
+      Scan Visor and After Omega Pirate Defeated
+  > Event (Omega Pirate Defeated)
+      X-Ray Visor and Have all Beams
+  > Pickup (Phazon Suit)
+      After Omega Pirate Defeated
 
 > Dock to Processing Center Access; Heals? False
-  * Normal Door to Processing Center Access/Dock to Elite Quarters
+  * Plasma Door to Processing Center Access/Dock to Elite Quarters
+  > Event (Omega Pirate Defeated)
+      X-Ray Visor and Have all Beams
+
+> Event (Omega Pirate Defeated); Heals? False
+  * Event Omega Pirate Defeated
+  > Dock to Elite Quarters Access
+      Trivial
+
+> Pickup (Phazon Suit); Heals? False
+  * Pickup 83; Major Location? True
+  > Dock to Elite Quarters Access
+      Trivial
 
 ----------------
 Central Dynamo
 Asset id: 4272124642
 > Dock to Dynamo Access; Heals? False
-  * Normal Door to Dynamo Access/Dock to Central Dynamo
+  * Ice Door to Dynamo Access/Dock to Central Dynamo
+  > Dock to Save Station Mines B
+      After Security Drone Defeated
+  > Event (Security Drone Defeated)
+      Any of the following:
+          X-Ray Visor
+          Combat (Beginner) and Invisible Objects (Beginner)
 
 > Dock to Quarantine Access A; Heals? False
-  * Normal Door to Quarantine Access A/Dock to Central Dynamo
+  * Ice Door to Quarantine Access A/Dock to Central Dynamo
+  > Dock to Save Station Mines B
+      After Security Drone Defeated and After Central Dynamo Bendezium Destroyed
+  > Event (Security Drone Defeated)
+      All of the following:
+          Space Jump Boots and After Central Dynamo Bendezium Destroyed
+          Any of the following:
+              X-Ray Visor
+              Combat (Beginner) and Invisible Objects (Beginner)
```

Comment:
Only needs [X-Ray or (Combat and Invisible Objects)]

Reply:
Actually it does need space jump boots, or at least it did when I tried it. The spawn trigger for the drone is actually on the ledge where you come in from Dynamo Access.

Reply:
I stand corrected then